### PR TITLE
Add integ tests

### PR DESCRIPTION
**Pull request by sarthak77**
Ref issue : https://github.com/hypertrace/query-service/issues/130

### CODE REVIEW

File: query-service/src/integrationTest/java/org/hypertrace/core/query/service/htqueries/HTPinotQueriesTest.java [hunk 1]
```diff
@@ -447,6 +447,11 @@ public void testAttributeExpressionQueries(
       throws IOException {
     return Stream.of(
         Arguments.arguments(buildQueryFromJsonFile("query1.json"), 10, "server"),
-        Arguments.arguments(buildQueryFromJsonFile("query2.json"), 2, "server"));
+        Arguments.arguments(buildQueryFromJsonFile("query2.json"), 2, "server"),
+        Arguments.arguments(buildQueryFromJsonFile("query3.json"), 1, "0.0"),
+        Arguments.arguments(
+            buildQueryFromJsonFile("query4.json"),
+            10,
+            "{\"internal.span.format\":\"jaeger\",\"http.status_code\":\"200\",\"component\":\"net/http\",\"span.kind\":\"server\",\"sampler.type\":\"const\",\"sampler.param\":\"1\",\"http.url\":\"/dispatch?customer=392&nonse=0.2664909002836733\",\"servicename\":\"frontend\",\"http.method\":\"GET\"}"));
```
Comment (kotharironak) [inline query-service/src/integrationTest/java/org/hypertrace/core/query/service/htqueries/HTPinotQueriesTest.java:455]: Can you also put that in json file as `query4-expected-response.json`? (its not redable)

Comment (sarthak77) [reply query-service/src/integrationTest/java/org/hypertrace/core/query/service/htqueries/HTPinotQueriesTest.java:455]: It is coming in this format only. It is a string.

Comment (sarthak77) [reply query-service/src/integrationTest/java/org/hypertrace/core/query/service/htqueries/HTPinotQueriesTest.java]: created a new test for this case.

File: query-service/src/integrationTest/java/org/hypertrace/core/query/service/htqueries/HTPinotQueriesTest.java [hunk 2]
```diff
@@ -419,6 +419,19 @@ public void testAttributeExpressionQueries(
...
+  public void testNotContainsKeyQuery() throws IOException {
+    LOG.info("Not contains key query");
+    Iterator<ResultSetChunk> itr =
+        queryServiceClient.executeQuery(
+            buildQueryFromJsonFile("query4.json"), TENANT_ID_MAP, 10000);
+    List<ResultSetChunk> list = Streams.stream(itr).collect(Collectors.toList());
+    List<Row> rows = list.get(0).getRowList();
+    assertEquals(10, rows.size());
+    assertEquals(
+        "\"http.status_code\":\"200\"", rows.get(0).getColumn(0).getString().split(",")[1]);
```
Comment (kotharironak) [inline query-service/src/integrationTest/java/org/hypertrace/core/query/service/htqueries/HTPinotQueriesTest.java:432]: Are all the rows are having the same values? Would this always be true that the first-row first column will have this value?

Comment (sarthak77) [reply query-service/src/integrationTest/java/org/hypertrace/core/query/service/htqueries/HTPinotQueriesTest.java:432]: >Are all the rows are having the same values?

No

>Would this always be true that the first-row first column will have this value?

Should be as we are generating same load ?

File: query-service/src/integrationTest/java/org/hypertrace/core/query/service/htqueries/HTPinotQueriesTest.java [hunk 3]
```diff
@@ -419,6 +419,19 @@ public void testAttributeExpressionQueries(
     assertEquals(expectedValue, rows.get(0).getColumn(0).getString());
   }
 
+  @Test
+  public void testNotContainsKeyQuery() throws IOException {
+    LOG.info("Not contains key query");
+    Iterator<ResultSetChunk> itr =
+        queryServiceClient.executeQuery(
+            buildQueryFromJsonFile("query4.json"), TENANT_ID_MAP, 10000);
```
Comment (rish691) [inline query-service/src/integrationTest/java/org/hypertrace/core/query/service/htqueries/HTPinotQueriesTest.java:427]: is there a 1-1 mapping between the test method and query file, if yes then maybe you can name the query file after the method name?

**Review by rish691 — APPROVED** [review]

**Pull request merged**